workspaces/ts-source-parser/package.json

```
@@ -3,13 +3,15 @@
   "version": "1.0.0",
   "description": "TypeScript parser for AST XRay analysis",
   "type": "module",
-  "exports": "./index.js",
-  "types": "./index.d.ts",
+  "exports": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "scripts": {
-    "lint": "eslint src test",
-    "test-only": "glob -c \"node --test-reporter=spec --test\" \"./test/**/*.spec.js\"",
+    "build": "tsc -p tsconfig.json",
+    "lint": "eslint . --ext .js,.ts",
```

```suggestion
    "lint": "eslint src test",
```
There is no `.js` anymore :p